docs/website/blog/2025-05-07-client-security-advisory.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Mithril Cardano database snapshots security advisory
+authors:
+  - name: Mithril Team
+tags:
+  [spo, mithril client, mainnet, production, beta, security, cardano-database]
+---
+
+### Mithril snapshots for Cardano database could be compromised by an adversary
+
+The Mithril team has published a [security advisory](https://github.com/input-output-hk/mithril/security/advisories/GHSA-qv97-5qr8-2266) for users running the Mithril client on the `mainnet` infrastructure:
+
+- **Identifier**: GHSA-qv97-5qr8-2266
+- **Title**: Mithril snapshots for Cardano database could be compromised by an adversary
+- **Location**: [GHSA-qv97-5qr8-2266](https://github.com/input-output-hk/mithril/security/advisories/GHSA-qv97-5qr8-2266)
+- **Severity**: Moderate (4.9/10).
+
+:::danger
+
+We strongly encourage all the `mainnet` users running a **client library or client CLI** to update to the latest versions to prevent the issue:
```

Comment:
```suggestion
We strongly encourage all `mainnet` users running a **client library or client CLI** to update to the latest versions to prevent the issue:
```